test/ui/contact_list/start_chat_bottom_sheet_test.dart

```diff
@@ -221,43 +197,19 @@ void main() {
         await tester.pump();
       }
 
-      testWidgets('hides loading indicator', (WidgetTester tester) async {
```

Comment:
I ended up deleing this tests, cause actually it's not just a test failure, the loadign shows when fetching the keypackage before throwing the error, and I think it's fine that is does